codecov.yml

```diff
@@ -6,6 +6,8 @@ coverage:
     patch:
       default:
         informational: true
+github_checks:
+  annotations: false
```

Comment:
So this line fixed the annotations (comments that litter the PR reviews)?!

Reply:
Yes, the docs say so. To actually test it we'll need code changes